.github/pull_request_template.md

```diff
@@ -7,18 +7,53 @@ v    If your PR doesn't close an issue, that's OK!  Just remove the Closes: #XXX
 
 Closes: #XXX
 
-## Description
+## What is the purpose of the change
 
-<!-- Add a description of the changes that this PR introduces and the files that
-are the most critical to review.
--->
+*Add a description of the overall background and high level changes that this PR introduces *
```

Comment:
Can we keep these as comments? I personally would prefer to avoid having to delete these lines on every PR. With comments, the PR author can keep them and just start typing below

Reply:
Hi Roman, the problem is, those comments won't show up when a PR is created, so submitters won't be able to see it at all. E.g. in https://github.com/osmosis-labs/osmosis/pull/1233, it just doesn't show up. 

Understand it takes a small effort to delete lines. But missing guidance can be quite confusing to new contributors like me who may not know what it's about.

Overall, hope it can offer a better practice to ask everyone, both existing and new contributors, to put more thoughts into PR formatting and content, that will eventually make us move faster as a group

Reply:
> those comments won't show up when a PR is created, so submitters won't be able to see it at all

Could you help me understand - what exactly do you mean by not being able to see them as a submitter?

Here's an attempt to create a test PR where the comments are visible:
![image](https://user-images.githubusercontent.com/34196718/163694204-6a0dc25f-d5eb-4a87-a0c2-5819c637de7c.png)

> But missing guidance can be quite confusing to new contributors like me who may not know what it's about.

Although I agree that these descriptions are valuable, I think they would be better kept as comments. The fundamental purpose of this text is for submitters to read them and understand what needs to be done. In my opinion, comments help to achieve that. In addition, I foresee future PRs not having these descriptions removed, making them less readable and more cluttered with text. With comments, the reviewers wouldn't see this detailed instruction text

Reply:
idk, I think they're better off visible as well, you'd want them to show when you hit preview for instance.

I also don't think this is a big deal to change once we try something initially. Perhaps we start with it visible, and if we observe its not causing the effect we want, we make them comments in a future PR?

Reply:
Discussed offline - let's make it visible and try :+1: